README.md

```diff
@@ -32,8 +33,7 @@ This process will open browser windows to interact with the DevCycle universal l
 To switch organizations once logged in, the [org command](docs/org.md) can be used.
 ## Using Client Credentials
 ### Credentials File
-Create a subdirectory inside the directory where you're running the CLI called `.devcycle`, then inside that directory
-create an `auth.yml` file with the following contents:
+Check the [oclif docs](https://oclif.io/docs/config) to find where the default `configDir` is located. Within that directory, create a `devcycle` directory, if it does not already exist. Inside of the `devcycle` directory, create an `auth.yml` file with the following contents:
```

Comment:
not sure if this is addressed by Laura's other comment, but it doesn't seem like we should need to point people to oclif to know where the configuration is stored, we should control it ourselves

Reply:
We're using oclif's built-in "configDir" as they recommend, which puts the configuration in the right place for user-specific configuration depending on what platform you're running the CLI on.